pkg/chaosdaemon/tasks/task_manager.go

```diff
@@ -37,13 +38,13 @@ type Recoverable interface {
 }
 
 // Creator init an Injectable with values.
-// We use it in a case that TaskConfig.data init an Injectable task here.
+// We use it in a case that TaskConfig.Data init an Injectable task here.
 type Creator interface {
```

Comment:
I think we can define an interface instead of `Creator`, `Addable`, `Object`, `Assign`, and `TaskExecutor`. 
```
type TaskExecutor interface {
	Merge() error 
        Assign() error 
        ...
}
```

Reply:
I split the interface because developer choose to implement some of these interface. If developer do not need some part of function , they do not need to inplement the base interface (sometimes they may not have the ability to implement it).
I think it is a kind of falling back ,if we merge the interface.